### PR TITLE
Hello

### PR DESCRIPTION
**Pull request by jucora**
Hello, this is my solo project about css gradients.

I really appreciate your feedback, it'll be so useful for me.

Thank you!!!

Bless

### CODE REVIEW

**Comment by mosaaleb** [comment]
Currently reviewing

File: index.html [hunk 1]
```diff
@@ -10,27 +10,65 @@
...
+					<li><a href="#">Store</a></li>
+					<li><a href="#">Mac</a></li>
+					<li><a href="#">iPod</a></li>
+					<li><a href="#">iPhone</a></li>
+					<li><a href="#">iPad</a></li>
+					<li><a href="#">iTunes</a></li>
+					<li><a href="#">Support</a></li>
+					<li style="padding-left: 15px; padding-right: 15px;">
+						<form action="/action_page.php" style="position: relative;">
+      						<input type="text" name="search" value="     ">
```
Comment (mosaaleb) [inline index.html]: It's perfectly [fine](https://stackoverflow.com/questions/3294572/is-input-well-formed-without-a-form) to use `<input>` without the need to include the `<form>` tag for simple components likes search field.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @jucora :wave:

### Great job :clap:. Kudos on using semantic html tags. The UI is clean and organized. You have already done most of the requirements, yet there is some minor requirements need to be handled first before moving on.


### Main content links (inside `<header>` tag)
- [x] There should be three links
- [x] Each link is an `<a>` HTML tag, with a `<div>` tag inside it.
- [x] The `<div>` tag  inside the `<a>` tag should use an image as background -- The text on the original page is actually an image.

[Why to use background image css property instead of `` html tag?](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image).

### Promo links
- [x] The top part is made of four `<a>` tags with `<div>` tags inside it
- [x] The `<div>`s inside the `<a>` tags should use an image as a background

The same concept of background-img vs `` tag also applies here.

### General
- [x] In your stickler file there is no need to enable rubocop as it is the linter for ruby code. We only need to enable style lint. Follow [this](https://github.com/microverseinc/linters-config/tree/master/css) for more information on how to setup it.


**[Optional]** : For more modularity on our code base you can use separate `reset.css` file instead of putting its code inside `style.css`. This will pave the way for modular css design when you work with multiple css files in the future.

Please submit a new code review once you met these requirements.

Feel free to contact me on slack if  you have any questions.
### Happy coding :computer:

**Comment by Oghenebrume50** [comment]
Hi, great work so far, just a few corrections left to do and you are good

File: styles.css [hunk 2]
```diff
@@ -1,81 +1,175 @@
...
-main{
-	background: url("url(/web/20140301004259im_/http://images.apple.com/global/nav/images/globalnav_text.svg)");
-	padding:0 12.5%;
+*{
+	font-family: 12px/5px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+body{
+	box-sizing:content-box;
+	padding-left: 12.5%;
+	padding-right: 12.5%;
```
Comment (Oghenebrume50) [inline styles.css]: These paddings make the whole body to the centralized, we do not want this

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]

File: styles.css (hunk 2 above)
Comment (Oghenebrume50) [reply styles.css]: Just correct the above and submit for re-review, whike you are at it check out Github flow and use it for your next project, if you do not understand send me a message on slack

**Comment by mcrd25** [comment]
# Currently Under Review by TSE

**Review by mcrd25 — APPROVED** [review]
# Congratulations! 🎉 🚀 

Hi @jucora 👋,

You've done a good job implementing this clone and have successfully implemented changed requested by previous reviwers. you may now merge to master.

Here are some things to consider moving forward:
- Use [GitHub Flow](https://guides.github.com/introduction/flow/) with appropriately names feature branch in remaining HTML/CSS projects. Here is a good article on differences between Git Flow and GitHub flow: https://lucamezzalira.com/2014/03/10/git-flow-vs-github-flow/
- consider making improvements to navbar in future so that it looks and feels closer to original. I really like how you hover effect has a gradient also, it is  a really cool addition. Well done! 😄 


***If you have any questions, now or in the future, do not hesitate to contact me on social media or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

**Pull request merged**